### PR TITLE
attempt at fixing solarpanel being disposalsv2

### PR DESCRIPTION
**Pull request by PaulRitter**
grafik

### CODE REVIEW

**Comment by 20kdc** [comment]
...question:
if this works, what even does EntityQuery *do*

File: Content.Server/Solar/EntitySystems/PowerSolarSystem.cs [hunk 1]
```diff
@@ -69,15 +70,32 @@ internal sealed class PowerSolarSystem : EntitySystem
...
+            SubscribeLocalEvent<SolarPanelComponent, ComponentShutdown>(OnShutdown);
 
             // Initialize the sun to something random
             TowardsSun = MathHelper.TwoPi * _robustRandom.NextDouble();
             SunAngularVelocity = Angle.FromDegrees(0.1 + ((_robustRandom.NextDouble() - 0.5) * 0.05));
         }
 
+        private void OnShutdown(EntityUid uid, SolarPanelComponent component, ComponentShutdown args)
+        {
+            _activePanels.Add(component);
```
Comment (20kdc) [inline Content.Server/Solar/EntitySystems/PowerSolarSystem.cs:91]: Nopenopenopenopenopenopenope

**Review by 20kdc — CHANGES_REQUESTED** [review]
Uh

File: Content.Server/Solar/EntitySystems/PowerSolarSystem.cs (hunk 1 above)
Comment (20kdc) [reply Content.Server/Solar/EntitySystems/PowerSolarSystem.cs:91]: ...in case you missed it: OnShutdown adding a panel to the active panels list

**Comment by PaulRitter** [comment]
shit anyways, i was optimizing the wrong part

**Comment by PaulRitter** [comment]
i should also go eat jfc did i really re-add it in shutdown

**Pull request closed without merging**